packages/nectar/src/sagas/errors/handleErrors/handleErrors.saga.test.ts

```diff
@@ -43,15 +40,16 @@ describe('handle errors', () => {
     userCertificate: ''
   }
 
-  test('receiving registrar server error results in retrying registration', async () => {
+  test('receiving registrar server error results in retrying registration and not putting error in store', async () => {
+    const errorPayload = {
+      community: community.id,
+      type: ErrorTypes.REGISTRAR,
+      code: ErrorCodes.NOT_FOUND,
+      message: ErrorMessages.REGISTRAR_NOT_FOUND
+    }
     await expectSaga(
       handleErrorsSaga,
-      errorsActions.addError({
-        community: community.id,
-        type: SocketActionTypes.REGISTRAR,
-        code: ErrorCodes.NOT_FOUND,
-        message: ErrorMessages.REGISTRAR_NOT_FOUND
-      })
+      errorsActions.handleError(errorPayload)
     )
       .withReducer(
```

Comment:
Our current pattern is to use real store object and nectar's reducers for sagas tests (see eg. sendMessage.saga.test)